### DISCUSSION TITLE
Start Guide

### DISCUSSION THREAD

**Discussion by aadhi-i** [MCA-CodeHub-MITS/members#7] [Announcements]
### Hey everyone,
**Feel free to ask any of your concerns here.**
1. Start a Q&A and clear your doubts.
2. Start General Discussions.
3. Bring up your ideas and make it live.
4. Having a confusion? Put it on poll.

### Anyone Having trouble with raising an issue or merging pull request contact the admins.

**Comment by akhilmohnn** [comment]
### Hey everyone! 👋
Excited to be part of this! Looking forward to learning and collaborating with all of you. Feel free to reach out if anyone needs help or wants to discuss ideas. Let's make this a great space for everyone! 🚀😊
